features/SideNavigation.feature

```diff
@@ -0,0 +1,165 @@
+            | Core Category Section                       | 003B45 (dark green) | FFFFFF              | bold       | FFFFFF      | 003B45 (dark green)                                   | N/A              | corresponding content page opening up |
+            | Core Category Section with navigation arrow | N/A                 | N/A                 | N/A        | N/A         | 003B45 (dark green)                                   | arrow point up   | N/A                                   |
+            | Parent                                      | 003B45 (dark green) | FFFFFF              | N/A        | N/A         | 003B45 (dark green)                                   | N/A              | corresponding content page opening up |
+            | Parent with dropdown arrow                  | FFFFFF              | 003B45 (dark green) | bold       | N/A         | 003B45 (dark green)                                   | arrow point up   | N/A                                   |
+            | Child                                       | 003B45 (dark green) | FFFFFF              | bold       | N/A         | dropdown line changes to 2px width and green (06966F) | N/A              | corresponding content page opening up |
+            | Child with dropdown arrow                   | FFFFFF              | 003B45 (dark green) | bold       | N/A         | 003B45 (dark green)                                   | arrow point up   | N/A                                   |
+
+    @ui
+    Scenario: When a page is not found a custom 404 error page is displayed
+        Given the user clicks on a link which is not available
+        Then they see a 404 Page Not Found message
+        And they see a custom design with Squatchy sitting in the middle of the 0
+        And a message:
+            """Please excuse the mess—We are in the process of making our documentation even better.
+    If you cannot find the information you're looking for, let our Success team know, and they will be more than happy to provide!"""
+
+    @ui
+    Scenario: The logo at the top of the side navbar takes you to the "https://docs.saasquatch.com/" page
+        Given the user is on any page of the docs portal
+        When they click the SaaSquatch logo at the top of the navigation bar
+        Then the navigation bar has every dropdown closed
+        And shows the 6 main menu items
+        And the central section of the page shows the "help center" search page
+        And the url is "https://docs.saasquatch.com/"
+
+    Scenario: There is a search box at the top of the side navbar (more specs available in a separate file for Search)
+        Given the user is on the SaaSquatch docs page at any location
+        Then a search input field is shown in the sidebar
+        And the default state for the search input field is empty
+        And the search is working expected
+
+    Scenario: In the side navbar there are different menu item types
```

Comment:
I'm not sure fully what this spec is trying to document or test, further up you establish that these different types exist - but this spec doesn't mention any specifics of these types